src/worker/WorkerDomNodeStub.js

```diff
@@ -1,12 +1,13 @@
 import Bridge from './WorkerBridge';
-import {CONSTRUCTOR, ADD_CHILD, ADD_CHILD_INDEX, REMOVE_CHILD, REMOVE_CHILD_INDEX, REPLACE_AT ,SET_CONTENT, REMOVE_EVENT_HANDLERS, SET_ATTRIBUTES, ADD_EVENT_HANDLERS, RENDER} from './../common/constants';
+import {CONSTRUCTOR, ADD_CHILD, ADD_CHILD_INDEX, REMOVE_CHILD, REMOVE_CHILD_INDEX, REPLACE_AT ,SET_CONTENT, REMOVE_EVENT_HANDLERS, SET_ATTRIBUTES, ADD_EVENT_HANDLERS, RENDER, INVOKE} from './../common/constants';
 
 var guid = 0;
 
 export default class WorkerDomNodeStub {
-    constructor(reactId, el, options) {
+    constructor(reactId, el, options, bridge) {
```

Comment:
Does bridge have to be passed in for each component type ? Can't it be something that can be set at the `render` method, and used globally for an application. 
Is this because you could have multiple channels in the same app?